project.godot

```diff
@@ -8,22 +8,40 @@
 
 config_version=4
 
-_global_script_classes=[  ]
+_global_script_classes=[ {
+"base": "Control",
+"class": "DialogueBox",
+"language": "GDScript",
+"path": "res://cutscene_engine/cutscene_scripts/DialogueBox.gd"
+}, {
+"base": "Node",
+"class": "DialoguePlayer",
+"language": "GDScript",
+"path": "res://cutscene_engine/cutscene_scripts/DialoguePlayer.gd"
+}, {
+"base": "Node",
+"class": "DialogueStarter",
+"language": "GDScript",
+"path": "res://cutscene_engine/cutscene_scripts/DialogueStarter.gd"
+} ]
```

Comment:
Can someone who knows more about Godot chime in on the consequences of this?

Reply:
It's not ideal, because it means this script will be loaded before every scene, even those which don't use this system. As long as we don't use this frequently it might be okay though, but the problem I foresee is if we add a ton of strings to the point where every scene becomes super laggy.